package.json

```diff
@@ -63,7 +63,7 @@
     "register": "ts-node src/commands/Register.ts",
     "start": "ts-node src/commands/Start.ts",
     "tdd": "npm run test -- --watch --watch-files src,test",
-    "test": "ALLOW_CONFIG_MUTATIONS=true  npx mocha -r ts-node/register --extensions ts 'test/**/*.{test,spec}.ts'"
+    "test": "ALLOW_CONFIG_MUTATIONS=true  npx mocha -r ts-node/register -r tsconfig-paths/register --extensions ts 'test/**/*.{test,spec}.ts'"
```

Comment:
Why this is necessary in this pr?

Reply:
This part is necessary to avoid imports using relative paths in tests (e.g.: import `src/myFile.ts` instead of `import ../../src/myFile.ts`)

Reply:
Ye i mean why in this pr and not before, for the whole project. Detail, to reduce deps and keep pr minimal

Reply:
Just because we didn't manage to find a way before. BTW I hear you so, if necessary, I could split this change into a different PR

Reply:
Nono. It us ok. For the record, i leave the comment